### DISCUSSION TITLE
Request Activity to Implement Deadline Schedule Notification

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4782] [Plugins]
![11](https://github.com/kimai/kimai/assets/165639563/5ccc004f-4911-4010-a6fb-33960d263f67)

Hello,

You want to add a deadline to your activities, use them as Sprint Memoirs, display a deadline to your project activities, enter no deadlines, or implement an email notification when you have a deadline.
Is it possible to ask for advice on code tasks that require you to add or change a deadline column in your activity entity to implement this feature?

**Comment by seleondevel** [comment]
I also need to implement an event about the Activity deadline in kimai src/Event, would it be difficult to add this part?

**Comment by seleondevel** [comment]
Could you please review the idea of adding a deadline registration and notification function to Kimai's Activities menu in the next version? Please answer that idea as well.

**Comment by seleondevel** [comment]
@kevinpapst I also want the functionality of the sprint work to be reflected.
I want to hear your thoughts.

**Comment by kevinpapst** [comment]
Use the Task planning plugin: https://www.kimai.org/store/task-management-bundle.html

**Comment by seleondevel** [comment]
![demo](https://github.com/kimai/kimai/assets/165639563/d38030eb-34b8-43b7-95d7-a95b98407ed6)
@kevinpapst  Thank you. I approached the demo and checked the task planning plug-in you recommended.
By the way, in the demo, there is a separate Task menu, and what I want is that there is a Task instead of Activity within the project.

Shouldn't we proceed with the task within the project? I wonder why it was organized separately from the project.

By any chance, is it possible to change this plug-in to create a task within the project? I'd like to hear your opinion on this part.

**Comment by seleondevel** [comment]
![스크린샷 2024-04-16 180603](https://github.com/kimai/kimai/assets/165639563/5d717069-9aa7-4ce8-bf8b-131a14aebe4e)

Oh! There was a task under the project. I will review it again! Thank you!
